src/components/datetime.js

```diff
@@ -13,6 +13,11 @@ module.exports = function(app) {
         controller: ['$scope', '$timeout', function($scope, $timeout) {
           if ($scope.builder) return;
 
+          // Close calendar pop up when tabbing off button
+          $scope.onKeyDown = function(event) {
```

Comment:
It's been a while but I'm sure I tried that and wasn't happy with the result.  Maybe because tabbing through to the drop down button closed the calendar before I thought it should.